<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>708</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="1">
     <widget class="QProgressBar" name="progressBar">
      <property name="value">
       <number>24</number>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="processLabel">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetNoConstraint</enum>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QToolButton" name="addFileButton">
          <property name="text">
           <string>Add directory</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="finderLabel">
          <property name="text">
           <string> Find:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit"/>
        </item>
        <item>
         <widget class="QPushButton" name="nextButton">
          <property name="text">
           <string>Next usage</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="foundAmountLabel">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="1" column="0" colspan="2">
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="MyTreeWidget" name="treeWidget">
        <property name="animated">
         <bool>true</bool>
        </property>
        <attribute name="headerMinimumSectionSize">
         <number>100</number>
        </attribute>
        <attribute name="headerStretchLastSection">
         <bool>false</bool>
        </attribute>
        <column>
         <property name="text">
          <string>Filename</string>
         </property>
         <property name="background">
          <color>
           <red>236</red>
           <green>232</green>
           <blue>173</blue>
          </color>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Directory</string>
         </property>
         <property name="background">
          <color>
           <red>229</red>
           <green>186</green>
           <blue>125</blue>
          </color>
         </property>
        </column>
       </widget>
       <widget class="TextViewer" name="textViewer"/>
      </widget>
      <widget class="DirList" name="dirWidget">
       <column>
        <property name="text">
         <string>Directory</string>
        </property>
       </column>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Fi&amp;le</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionScan_Directory">
   <property name="text">
    <string>&amp;Scan Directory...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MyTreeWidget</class>
   <extends>QTreeWidget</extends>
   <header>mytreewidget.h</header>
  </customwidget>
  <customwidget>
   <class>TextViewer</class>
   <extends>QTextEdit</extends>
   <header>textviewer.h</header>
  </customwidget>
  <customwidget>
   <class>DirList</class>
   <extends>QTreeWidget</extends>
   <header>dirlist.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
